### ISSUE TITLE
Surplus Addenda at Corrigenda Pages (MD)

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/CORRECTIONS#14]
@funderburkjim I understand that starting from `1308 hvala , Suppl.` https://archive.org/stream/sanskritenglishd00moniuoft#page/1308/mode/2up everything is incorporated since 2012, right? But are the 2 additional pages from the 1960 reprint as well? I checked a few and did not notice the changes. Otherwise `asOnAman` should be deleted or at least marked for deletion, right?

![asau](https://cloud.githubusercontent.com/assets/80761/4638106/8aa6f4aa-53f5-11e4-83af-80fcdb36d251.PNG)

P.S. http://www.sanskrit-lexicon.uni-koeln.de/scans/csldoc/dictionaries/prefaces/mwpref.html abbreviation page missing (because pages 2-6 of mw_bookmark.pdf are left out).

**Comment by gasyoun** [comment]
@funderburkjim question remains open about MW.

**Comment by funderburkjim** [comment]
@gasyoun What do you suggest be done?

**Comment by gasyoun** [comment]
@funderburkjim is my suggestion right? Are supplements integrated?

**Comment by funderburkjim** [comment]
Yes -  but I am not sure whether you have discovered some further supplement pages that were not included among the supplements that have been integrated. 
The supplement pages that were integrated can be seen [here](http://www.sanskrit-lexicon.uni-koeln.de/scans/MWScan/index.php?sfx=pdf).

Are the supplements from your archive.org reference included in these Cologne images?

**Comment by gasyoun** [comment]
@funderburkjim supplements you linked to are part of the original book. Supplements I talked about were supplements of supplements - 2 pages (Addenda et Corrigenda) in the beginning of the newest edition. I could not trace it's implementation on a single example.
https://archive.org/stream/sanskritenglishd00moniuoft#page/n5/mode/2up
https://archive.org/stream/sanskritenglishd00moniuoft#page/n7/mode/2up

**Comment by funderburkjim** [comment]
These pages from archive.org definitely do NOT appear among the scanned images for MW at Cologne.

And these pages from archive.org are not part of the material received from Thomas in 2010 as additions,
which refer only to the material at the end of the book.

So, AFAIK, there is no digitization of these archive.org pages.

**Comment by gasyoun** [comment]
@funderburkjim do you think we should ask Thomas about necessity of it's inclusion?

**Comment by funderburkjim** [comment]
I've sent email question to Thomas.

**Comment by drdhaval2785** [comment]
@funderburkjim any update from Thomas ?

**Comment by gasyoun** [comment]
I can't open archive.org anymore, to check if it's still there. Need to extract both pages and add them here.

**Comment by funderburkjim** [comment]
@drdhaval2785   I've basically given up trying to contact Thomas, due to not getting responses back from my emails to him.  I suspect he basically is on to something else in his life.  In particular, I never got a reply back from above.   

To incorporate these  alleged extra pages into the Cologne MW digitization would require entering 'by hand', AND (possibly the hardest part) deciding what the L-number should be.  

In looking at the archive org link,  in particular at 'anupaskfta'  and comparing to current digitization,
there are points of confusion.  The link says 'Add blameless after disinterested'.  However Digitization already shows 'blameless' and does not show 'disiniterested'.  
link:
![image](https://cloud.githubusercontent.com/assets/6393033/10893623/d75a5138-8177-11e5-9548-69138523e321.png)

Cologne:
![image](https://cloud.githubusercontent.com/assets/6393033/10893652/ff78e92c-8177-11e5-8fea-bc3b5189a2cd.png)

Note the difference in how the vocalic r  (= slp 'f') is represented.  To me, these differences raise questions of provenance of the Archive Org edition, even though both it and the edition used for Cologne digitization both claim to be reprints of the 1899 edition.

**Comment by gasyoun** [comment]
So Thomas is gone with the winds. If we would only knew who is the author of the 2nd addition of supplements. What you've documented show that not everything is possible to implement, because wrong corrections are proposed. I've seen such cases, but not all of them are such. I guess entering 2 pages we can manage - what about the L business? Any hints? Or first we need to find if there are new meanings in old headwords and only after can decide?

**Comment by funderburkjim** [comment]
1. Can you determine, by comparing the front matter of the archive.org edition and the front matter of the Cologne edition,  what is the relation between the two editions?

2. Regarding the 'L-business'  (and also the 'H-codes (H1, H1A, H1B, H1C, H2, etc.)  these would have to be determined in a 'second phase'.   The first phase would be typing (in a form consistent with the markup of monier.xml) the given data.   If you plan to actually do this, we can work out the details involved in the first phase.

**Comment by gasyoun** [comment]
To answer your question I need to see the file again. In Russia archive.org has been banned lately, so I asked @Shalu411 but her inet is down, so we'll have to wait. I want to include the 3 pages here. As per the editions - there is a bad practice of reprints of English books in India not to explain what is the difference if any. It's not so in Russia - we explain and document, there is a tradition for it. So in this case I can add, that the 3 additional pages is the only difference.
I'm thinking of doing this, but can't see where to start.

**Comment by funderburkjim** [comment]
@gasyoun   Here is curl command, which works in Git Bash:

```
curl -o archiveorgmw.pdf https://ia801406.us.archive.org/27/items/sanskriteng
lishd00moniuoft/sanskritenglishd00moniuoft.pdf
```

I'm also downloading, and if the 'curl' is also blocked for you, I will post the download to 
dropbox --  let me know if this posting to dropbox is needed by you.

**Comment by gasyoun** [comment]
@funderburkjim thanks, as you see above I allready posted screenshots of the needed pages. After a discussion at https://groups.google.com/forum/#!topic/nagari/MV-HcsNX40I I realized it's MD, not MW. Corrections of MD, the pages have been wrongly added to a MW scan.

**Comment by funderburkjim** [comment]
I joined the above Google Group so that I could read the discussion (Using Google to translate the Russian in the discussion).

The entry for 'an-upa-skfta'  provides strong evidence that the posted screenshots are from an edition of MD, as you concluded, since the terminology is identical in the Cologne digitization under anupaskfta.

Another similar case is aDyUQa, which appears with same wording in both posted screenshots and in Addenda of Cologne edition.

However, examples of aNkola and atikopana  from posted screenshots do not appear in Cologne edition.   So the posted screenshots are from a different edition, which I speculate would be a later edition than the Cologne edition.

**Comment by gasyoun** [comment]
There might be two editions of MD, indeed. Before I was not aware of them and believed only in one. The 1956 and 1960 editions are pure reprints. The roots must go deeper.
The Google group goes back to 2005 - the year I started Sanskrit NLP.

**Comment by gasyoun** [comment]
**Question type:** editorial-policy / text-variant

**Evidence found:**
The thread itself contains the key findings: the 2-page 'Addenda et Corrigenda' from the archive.org edition of MW (pages n5, n7) are **not** in the Cologne digitization. The final comment (2015-11-07) by @gasyoun confirmed these pages are from an MD (Macdonell) edition, not MW — wrongly bound into the archive.org MW scan. @funderburkjim identified `aNkola` and `atikopana` from those pages as absent from Cologne MD, suggesting a later edition.

**Assessment:**
Partially answerable: the archive.org Addenda pages are from a different (later?) edition of MD, not from MW. The Cologne digitization used a different edition. Whether those extra Addenda should be incorporated into Cologne MD requires verifying: (1) which MD edition Cologne used, (2) whether the aNkola/atikopana additions are authentic to that edition. The data entry and L-number assignment work was estimated feasible but not started.

**Suggested next step:** Determine which edition of MD the Cologne digitization is based on (check MD preface/meta file), then compare against the archive.org Addenda page entries to decide if any are missing from csl-orig/v02/md/md.txt.
